config/nf-core-defaults.config

```diff
@@ -29,6 +29,7 @@ process {
     }
     withLabel: process_long {
         time   = 20.h
+        cpus   = 4
```

Comment:
Don't forget to move this to functional_annotation.config so it keeps the nf-core defaults the same as nf-core settings